### PR TITLE
User with manager perms can change to all states.

### PR DESCRIPTION
**Pull request by reiterl**
Update `motion.set_state`.
Use `motion.can_manage_metadata`
Add and update tests.

### CODE REVIEW

**Comment by r-peschke** [comment]
Before approving the requirements should be cleared: which right allows to change the state without evaluation by next/prev-states. And this should be documented in https://github.com/OpenSlides/OpenSlides/wiki/motion.set_state.

File: tests/system/action/motion/test_follow_recommendation.py [hunk 1]
```diff
@@ -76,46 +76,6 @@ def test_follow_recommendation_correct(self) -> None:
         assert model.get("state_extension") == "test_test_test"
         assert model.get("last_modified", 0) >= check_time
 
-    def test_follow_recommendation_not_neighbour(self) -> None:
```
Comment (jsangmeister) [inline tests/system/action/motion/test_follow_recommendation.py:79]: Keep this test and fix `motion.follow_recommendation`: There, the check for the next states should still exist.

**Review by jsangmeister — CHANGES_REQUESTED** [review]

**Comment by jsangmeister** [comment]
Updated the wiki.

File: tests/system/action/motion/test_follow_recommendation.py (hunk 1 above)
Comment (reiterl) [reply tests/system/action/motion/test_follow_recommendation.py]: Readded the test and modify follow_recommendation and set_state, that the test passes.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**